src/main/java/homework_4/customFileReader/CustomFileReader.java

```diff
@@ -0,0 +1,71 @@
+package homework_4.customFileReader;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Scanner;
+
+public class CustomFileReader {
+
+    private String filename;
+    private Path path;
+
+    private final StringBuilder result = new StringBuilder();
+
+    public CustomFileReader(String filename) {
```

Comment:
here again:
when i see `filename` variable i expect it's the only file name, but no path.